day_7/fizzbuzz.rb

```diff
@@ -0,0 +1,19 @@
+number = Array(1..100)
+
+number.each do |x|
+  if x % 3 == 0
+    puts "fizz"
+  end
+
+
+  if x % 5 == 0
+    puts "buzz"
+  end
+
+  if x % 3 == 0 && x % 5 == 0
+    puts "fizzbuzz"
+  end
```

Comment:
It's an antipattern here to disconnect your conditional statements when considering multiple scenarios for x. Also, what would happen when x == 15?